src/nft/components/explore/Banner.tsx

```diff
@@ -0,0 +1,195 @@
+import clsx from 'clsx'
+import { Box } from 'nft/components/Box'
+import { Center, Column, Row } from 'nft/components/Flex'
+import { VerifiedIcon } from 'nft/components/icons'
+import { bodySmall, buttonMedium, header1, section } from 'nft/css/common.css'
+import { vars } from 'nft/css/sprinkles.css'
+import { fetchTrendingCollections } from 'nft/queries'
+import { TimePeriod, TrendingCollection } from 'nft/types'
+import { formatEthPrice } from 'nft/utils/currency'
+import { putCommas } from 'nft/utils/putCommas'
+import { formatChange, toSignificant } from 'nft/utils/toSignificant'
+import { ReactNode, useEffect, useReducer, useState } from 'react'
+import { useQuery } from 'react-query'
+import { Link } from 'react-router-dom'
+
+import * as styles from './Explore.css'
+
+const Banner = () => {
+  /* Sets initially displayed collection to random number between 1 and 5  */
+  const [current, setCurrent] = useState(Math.floor(Math.random() * 5 + 1))
+  const [hovered, toggleHover] = useReducer((state) => !state, false)
+  const { data: collections } = useQuery(
+    ['trendingCollections'],
+    () => {
+      return fetchTrendingCollections({ volumeType: 'eth', timePeriod: TimePeriod.OneDay, size: 5 })
+    },
+    {
+      refetchOnReconnect: false,
+      refetchOnWindowFocus: false,
+      refetchOnMount: false,
+    }
+  )
+
+  useEffect(() => {
+    /* Rotate through Top 5 Collections on 15 second interval */
+    const interval = setInterval(async () => {
+      if (collections) {
+        const nextCollectionIndex = (current + 1) % collections.length
+        if (!hovered) setCurrent(nextCollectionIndex)
+      }
+    }, 15000)
+    return () => clearInterval(interval)
+  }, [current, collections, hovered])
+
+  return (
+    <Box className={styles.fullWidth} onMouseEnter={toggleHover} onMouseLeave={toggleHover} cursor="pointer">
```

Comment:
This can get into the inverse state.
It's safer to define `hovered` using `useState` and explicitly call `setHover(true)`/`setHover(false)`.